packages/dds/tree/src/test/simple-tree/objectNode.spec.ts

```diff
@@ -63,6 +108,8 @@ describeHydration(
 			describe("constructor", () => {
 				it("empty", () => {
 					class Schema extends schemaFactory.object("x", {}) {}
+					type xxx = typeof Schema.info;
```

Comment:
Nit: can we be a bit more semantic with our names here? Will make future maintenance easier.

Reply:
Oops, that was left over from experiments that ended up in the other test. Removed.